acestep/acestep_v15_pipeline.py

```
@@ -173,6 +174,41 @@ def main():
+                args.device = "cuda:0"
+
+            if _multi_gpu_active:
+                # With separate GPUs, CPU offloading is unnecessary
+                if args.offload_to_cpu:
+                    args.offload_to_cpu = False
+                    print("  Multi-GPU: Disabled CPU offloading (models on separate GPUs)")
+        else:
+            # Single GPU: lm_device falls back to main device
+            if args.lm_device == "auto":
+                args.lm_device = args.device
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)